paymentsheet/src/test/java/com/stripe/android/paymentsheet/addresselement/AddressElementActivityContractTest.kt

```diff
@@ -14,16 +12,9 @@ class AddressElementActivityContractTest {
 
     @Test
     fun `AddressElementActivityContract args parcelize correctly`() {
```

Comment:
```suggestion
    fun `AddressLauncherActivityContract args parcelize correctly`() {
```

Does this file need to be renamed?

Reply:
spoke offline. Will hold off large scale renames until we finalize the API.